### ISSUE TITLE
Connected Links sometimes not attached to port of custom nodes

### ISSUE THREAD

**Issue by IbrahimTimimi** [Blazor-Diagrams/Blazor.Diagrams#150]
I designed my own custom nodes that inherits the `NodeModel` class. My plan is to make a simple workflow. 

Sometimes when I move the custom nodes around, the links appear not attached to the node's port as if not connected as shown in the photo below. How can I solve this issue to make the links always attached to the port?

![Capture12](https://user-images.githubusercontent.com/53964046/138804288-6f23bef8-612f-48b7-9ce6-21a9ada008c7.PNG)

**Comment by b43r** [comment]
I have the same issue with latest version 2.1.6. In my case this happens when I open a menu in the top of the page that shifts the whole canvas down.

**Comment by zHaytam** [comment]
How does the menu open?

**Comment by zHaytam** [comment]
Are you still having the issue?

**Comment by rob-ack** [comment]
@zHaytam Yes i can confirm that this issue still exists in the current version.

**Comment by zHaytam** [comment]
Hello, the GIF is very blurry so I can't really see anything, unfortunately.
Could you create a very simple/basic project to reproduce the issue? Then I can debug it and fix it. Because I need to know exactly what happens to the layout when you guys do what you do.
Thank you!

**Comment by lantingshangyue** [comment]
Hello ,I also meet it. when path overflow 
![R)IH9_WIRVR7B`T~)9~A_)3](https://user-images.githubusercontent.com/6137366/173111963-b7fcd8c2-fc2c-4711-b358-abd3890b8492.png)
![SI7L~RFNDQYM0$5E2VWAB$B](https://user-images.githubusercontent.com/6137366/173112545-2c759733-5598-449d-b354-44ee0b89ca65.png)

**Comment by lantingshangyue** [comment]
@zHaytam

**Comment by zHaytam** [comment]
Hello, would it be possible to have a runnable simple project where the issue show?
I really need one of you to create a project for me to run and do tests on, because I can't know the issue from pictures/GIFs unfortunately.

**Comment by lantingshangyue** [comment]
this is ok . in  .diagram-svg-layer style  add attribute [overflow: visible !important;]
